### PR TITLE
Implement #77

### LINKED ISSUE #77: Wrong cursor color when going back from class browsing view
When I browse a class by pressing the right-arrow key and then go back to the main result list by pressing the left-arrow key the cursor is red.

### PR DESCRIPTION
**Pull request by robertkirsten**
I changed the paragraph initialization, the color is now set right away when the paragraph is created.  This makes changing the cursorColor after a keyStroke unnecessary and also works when calling the parentList from a childList (the main issue here).

### CODE REVIEW

**Comment by BenBals** [comment]
Could you merge development into this branch to resolve merge conflicts?

File: packages/AlgernonTests.package/ALGTestGeneral.class/instance/testCursorColorAfterReturningFromChildList.st [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	|searchTerm field|
+	searchTerm := 'newParagraph'.
+	self runQuery: searchTerm.
+	self moveCursorTo: 'Class'.
+	self algerInstance showChildList.
+	self algerInstance showParentList.
+	
+	field := Algernon currentInstance textField.
+	self assert: (field paragraph caretColor = field class cursorColor).
+	Algernon close.
```
Comment (BenBals) [inline packages/AlgernonTests.package/ALGTestGeneral.class/instance/testCursorColorAfterReturningFromChildList.st]: You do not need to do this. This should be done via tearDown.

File: packages/AlgernonTests.package/ALGTestGeneral.class/instance/testCursorColorAfterReturningFromChildList.st [hunk 2]
```diff
@@ -0,0 +1,13 @@
+testing
+testCursorColorAfterReturningFromChildList
+
+	|searchTerm field|
+	searchTerm := 'newParagraph'.
```
Comment (BenBals) [inline packages/AlgernonTests.package/ALGTestGeneral.class/instance/testCursorColorAfterReturningFromChildList.st]: I dont see why this needs to be a variable.

File: packages/Algernon.package/ALGInputField.class/instance/createParagraph.st [hunk 3]
```diff
@@ -0,0 +1,9 @@
+initialization
+createParagraph
+
```
Comment (BenBals) [inline packages/Algernon.package/ALGInputField.class/instance/createParagraph.st:3]: This much whitespace is uncommon for smalltalk.

**Review by BenBals — APPROVED** [review]
Looks good to me. Just a few minor style issues.

**Comment by BenBals** [comment]
Pulling from dev should also fix the test.

**Comment by BenBals** [comment]
What makes this pr a draft? It looks good to me.

**Comment by robertkirsten** [comment]
No I messed something up when merging and now I can't fix it because I can't get rid of the wasChanged error

**Comment by BenBals** [comment]
Try to run
```
ALGInstallation instance cleanup.
ALGCore allSubInstancesDo:[:each | each delete].
ALGInstallation instance cleanup.
ALGCore allSubInstancesDo:[:each | each delete].
ALGBreadcrumbs allInstances do: [:each | each controller: nil. each abandon ].
ALGBackground allInstances do: [:each | each controller: nil. each abandon ].
```

**Comment by robertkirsten** [comment]
Unfortunately this does not help.

**Review by BenBals — COMMENTED** [review]
Looks still good.

**Pull request merged**